ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/package.json

```diff
@@ -26,16 +26,16 @@
     "react": "16.10.0",
     "react-dom": "16.10.0",
     "react-flow-renderer": "^10.3.17",
-    "react-router-dom": "5.3.4",
+    "react-router-dom": "5.0.0",
```

Comment:
Please pull this file from the repo and test with the package as updated for security fixes. 

As this pull request regresses those fixes.

Reply:
This also applies to the package-lock.json file.

Reply:
Just tested and it works with both the react-router-dom "5.3.4" & "5.0.0"